### ISSUE TITLE
[Enhancement] pattern based trash-empty

### ISSUE THREAD

**Issue by xxjapp** [andreafrancia/trash-cli#124]
For example trash-empty --pattern '/tmp/*.log'

**Comment by andreafrancia** [comment]
Just use:

```
trash-rm  '/tmp/*.log'
```

**Comment by xxjapp** [comment]
Thank you for your answer.

But the following command did nothing in my test. 

    trash-rm  '/tmp/*.log'

It seems that the command `trash-rm` /tmp/*.log files, but did not.

What I want is not to `trash-rm` /tmp/*.log files, but to `trash-empty` /tmp/*.log files already in trash.

**Comment by andreafrancia** [comment]
trash-rm deletes files from trash. So it works like trash-empty but it takes a pattern.
The problem is that it recognizes simple pattern like `*.log` but it does not recognize patterns like `/tmp/*.log`.

I'll check if it possible to fix it in order to make it recognize also patterns like `/tmp/*.log`

**Comment by xxjapp** [comment]
@andreafrancia 

Thank you for your great work.

**Comment by andreafrancia** [comment]
Fixed in version 0.21.5.22
```
$ touch /tmp/foo.log
$ touch /tmp/bar.log
$ trash-put /tmp/foo.log /tmp/bar.log
$ trash-list | grep log
2021-05-22 14:59:05 /private/tmp/bar.log
2021-05-22 14:59:05 /private/tmp/foo.log
$ trash-rm '/private/tmp/*.log'
$ trash-list | grep log
$ 
```

**Comment by andreafrancia** [comment]
@jaker-dotcom 
trash-rm uses fnmatch.fnmatchcase to match patterns, see https://docs.python.org/3/library/fnmatch.html for more details

I have just added this message to the usage of trash-rm:
```
$ trash-rm                                                                                                                                      ~/personal/trash-cli  master@0764e29
Usage:
    trash-rm PATTERN

Please specify PATTERN.
trash-rm uses fnmatch.fnmatchcase to match patterns, see https://docs.python.org/3/library/fnmatch.html for more details.
```

To see it you may want install trash-cli from source (`pip install git+https://github.com/andreafrancia/trash-cli`) or wait the next release.
